volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -0,0 +1,220 @@
+# This file is Copyright 2021 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+# Author: Gustavo Moreira
+
+import logging
+from collections import namedtuple
+from typing import Tuple, List, Iterable, Union
+
+from volatility3.framework import renderers, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import plugins
+from volatility3.plugins.linux import pslist
+
+vollog = logging.getLogger(__name__)
+
+MountInfoData = namedtuple("MountInfoData", ("mnt_id", "parent_id", "st_dev", "mnt_root_path", "path_root",
+                                             "mnt_opts", "fields", "mnt_type", "devname", "sb_opts"))
+
+class MountInfo(plugins.PluginInterface):
+    """Lists mount points on processes mount namespaces"""
+
+    _required_framework_version = (2, 1, 0)
+
+    _version = (1, 0, 0)
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        return [
+            requirements.ModuleRequirement(name="kernel", description="Linux kernel",
+                                           architectures=["Intel32", "Intel64"]),
+            requirements.PluginRequirement(name="pslist",
+                                           plugin=pslist.PsList, version=(2, 0, 0)),
+            requirements.ListRequirement(name="pids",
+                                         description="Filter on specific process IDs.",
+                                         element_type=int,
+                                         optional=True),
+            requirements.ListRequirement(name="mntns",
+                                         description="Filter results by mount namespace. "
+                                                     "Otherwise, all of them are shown.",
+                                         element_type=int,
+                                         optional=True),
+            requirements.BooleanRequirement(name="mount-format",
+                                            description="Shows a brief summary of the mount points information "
+                                            "with similar output format to the older /proc/[pid]/mounts or the "
+                                            "user-land command 'mount -l'.",
+                                            optional=True,
+                                            default=False),
+        ]
+
+    @classmethod
+    def _do_get_path(cls, mnt, fs_root) -> Union[None, str]:
```

Comment:
It's very good. I think explicit typing (argument type, return type) of methods is not essential, but it will reduce the time of good code and contributors. If you have time, could you type the rest of the methods below for uniformity? This is pretty NIT, so you can just resolve it.